### DISCUSSION TITLE
How To pass go struct and access fields from lua script

### DISCUSSION THREAD

**Discussion by sambestsolutions** [yuin/gopher-lua#383] [Q&A]
Please Help I want to pass go struct to a lua script and access struct feilds from lua script, Can anyone help with this
Note- Basically I want to pass struct to a lua script function with "CallByParam" 

Highly appreciate your help.

**Comment by sambestsolutions** [comment]
Can someone help with this?

**Comment by ZenLiuCN** [comment]
you may check i just done [glu](https://github.com/ZenLiuCN/glu)
